guides/source/action_controller_overview.md

```diff
@@ -899,6 +899,61 @@ Rails collects all of the parameters sent along with the request in the `params`
 [`query_parameters`]: https://api.rubyonrails.org/classes/ActionDispatch/Request.html#method-i-query_parameters
 [`request_parameters`]: https://api.rubyonrails.org/classes/ActionDispatch/Request.html#method-i-request_parameters
 
+#### `request.variant`
+
+Controllers might need to tailor a response based on context-specific information in a request. For example, controllers responding to requests from a mobile platform might need to render a HTML, JSON, or XML different content than requests from a desktop browser. They can add that context to the requests variant so that views render appropriately. Variant names are arbitrary, and can communicate anything from the request's platform (`:anrdoid`, `:ios`, `:linux`, `:macos`, `:windows`) to its browser (`:chrome`, `:edge`, `:firefox`, `:safari`), to the type of user (`:admin`, `:guest`, `:user`).
```

Comment:
```suggestion
Controllers might need to tailor a response based on context-specific information in a request. For example, controllers responding to requests from a mobile platform might need to render different content than requests from a desktop browser. One strategy to accomplish this is by customizing a request's variant. Variant names are arbitrary, and can communicate anything from the request's platform (`:anrdoid`, `:ios`, `:linux`, `:macos`, `:windows`) to its browser (`:chrome`, `:edge`, `:firefox`, `:safari`), to the type of user (`:admin`, `:guest`, `:user`).
```

There's two suggestions in here:
- I think the first sentence ended up containing the text of two different revisions, so I removed one
- I tweaked the middle sentence to transition from the why to the how a little smoother